server/views/applications/show.njk

```diff
@@ -26,7 +26,12 @@
 
       {{ showErrorSummary(errorSummary) }}
 
-      <h2 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy-status>Referral {{ taskList.status }}</h2>
+      {% if taskList.status === 'complete' %}
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy-status>Submit your referral below</h2>
```

Comment:
Do we need a test for this bit?

Reply:
I'd opt to not on this occasion, as it's quite low risk.